content/plugins/descriptions/security-insights.yaml

```diff
@@ -0,0 +1,94 @@
+# The server must be restarted to pick up changes in ths file.
+
+# Required. Must be unique across all files in the directory. No whitespace. Use hyphens as
+# separators.
+name: security-insights
+humanName: Security Insights
+heading: 'Backstage Security Insights Plugin'
+# Keep it short
+lead: 'See Security Insights for your components in Backstage'
+
+seo:
+  # Don't forget to end with "| Roadie"
+  title: 'Backstage Security Insights Plugin | Roadie'
+  description: |
+    See Security Insights in Backstage.
+
+logo:
+  # This must be a relative path. It should start without a slash or with ./
+  # The overall size of the file should be approximately 200 by 200.
+  # The image in the file should be approximately 100 by 100 pixels. It needs space around it.
+  # The image will be made greyscale by Gatsby Image Sharp.
+  fileSystemPath: './content/assets/logos/github/PNG/GitHub-Mark-Light-120px-plus.png'
+  # The dimensions are required. They are the outer dimensions of the image file.
+  width: 120
+  height: 120
+
+coverImage:
+  fileSystemPath: './content/assets/roadie-backstage-security-plugin.jpg'
+  alt: 'A screenshot of the Security Insights plugin. It is showing a security insights for a sample component.'
+
+# Instructions for someone who wants to use this plugin.
+# languages used here must be listed in the .babelrc
+gettingStarted:
+    # What will this step accomplish?
+  - intro: Install the plugin into Backstage.
+    language: bash
+    code: 'yarn add @roadiehq/backstage-plugin-security-insights'
+  - intro: Import it into your Backstage application.
+    language: typescript
+    code: |
+      // packages/app/src/plugins.ts
+      export { plugin as SecurityInsights  } from '@roadiehq/backstage-plugin-security-insights';
+  - intro: Add plugin API to your Backstage instance.
+    language: typescript
+    code: |
+      // packages/app/src/components/catalog/EntityPage.tsx
+      import { Router as SecurityInsightsRouter  } from '@roadiehq/backstage-plugin-security-insights';
+
+      const ServiceEntityPage = ({ entity }: { entity: Entity }) => (
+        ...
+        &lt;EntityPageLayout>
+          &lt;EntityPageLayout.Content
+            path="/security-insights"
+            title="Security Insights"
+            element={&lt;SecurityInsightsRouter entity={entity} />}
+          />
+          &lt;/EntityPageLayout>
+        &lt;/EntityPageLayout>
+      )
+  - intro: Run backstage app with and navigate to services tabs.
```

Comment:
Typo here? `Run backstage app with and...` something is missing

Reply:
Or should it be "Run the backstage app with the following command and navigate to the services tab"?